pages/Stats.vue

```diff
@@ -0,0 +1,282 @@
+<template>
+  <div
+    id="stats-viewer"
+    class="mx-4 mt-4 flex h-[91vh] flex-col"
+  >
+    <div class="stats-header">
+      <h1 class="text-2xl font-bold text-gray-700">
+        Statistics
+      </h1>
+    </div>
+    <div class="flex items-center justify-between">
+      &nbsp;
+      <div class="font-semibold text-center">Total Examined Names: {{ numRecords }}</div>
+      <div class="flex items-center space-x-2">
+        <div class="checkbox-wrapper">
+          <input
+            id="stats-checkbox"
+            v-model="myStats"
+            type="checkbox"
+          >
+          &nbsp;<label for="stats-checkbox" class="font-bold" >My Stats</label>
+        </div>
+        <div class="timespan-input  align-baseline">
+          <label
+            for="timespan"
+            class="font-bold align-baseline"
+          >Hours:&nbsp;</label>
+          <TextInput
+            id="timespan"
+            ref="numberinput"
+            v-model="timespan"
+            type="number"
+            style="width: 100px; margin-right: 10px;"
+          />
+          <button
+            class="bg-blue-800 text-white font-bold py-2 px-4 rounded"
+            @click="fetchStats"
+          >
+            Get Stats
+          </button>
+        </div>
+      </div>
+    </div>
+
+    <div class="h-full overflow-x-auto rounded-md border">
+      <table class="min-h-fit min-w-full table-auto">
+        <thead class="sticky top-0">
+          <tr class="h-12 bg-bcgov-blue5 text-left text-sm text-white">
+            <th
+              v-for="(header, index) in fields"
+              :key="index"
+              class="border-b border-gray-200 px-2 py-1"
+            >
+              {{ header.label }}
+            </th>
+          </tr>
+        </thead>
+        <tbody class="text-sm">
+          <tr
+            v-for="(item, index) in statsData"
+            :key="index"
+            class="align-top transition duration-200 ease-in-out border-b border-gray-300 hover:bg-gray-200"
+          >
+            <td class="whitespace-pre-line border-b border-gray-300 px-2 py-2">
+              <div class="container-fluid">
+                <div class="layout">
+                  <div
+                    class="flex fw-700 fs-15 my-2"
+                    style="font-weight: 600"
```

Comment:
I don't know if `fw` or `fs` are utility classes in Tailwind.
```suggestion
                    class="flex my-2 font-bold"
```